frontend/src/pages/TestFileManagement.tsx

```diff
@@ -0,0 +1,113 @@
+import { useAxiosApi } from 'hooks/pims-api';
+import { DocumentDetail, DocumentQueryResult, FileDownload } from 'models/api/DocumentManagement';
+import { ExternalResult } from 'models/api/ExternalResult';
+import { useCallback, useEffect, useState } from 'react';
+import { Button, Col, Container, Row } from 'react-bootstrap';
+import { FaDownload, FaUpload } from 'react-icons/fa';
+import { formatApiDateTime } from 'utils';
+
+export const TestFileManagement: React.FunctionComponent = () => {
+  const api = useAxiosApi();
+
+  const [documentList, setDocumentList] = useState<
+    DocumentQueryResult<DocumentDetail> | undefined
+  >();
+
+  const [selectedFile, setSelectedFile] = useState<File | null>(null);
+
+  const retrieveDocumentList = useCallback(async () => {
+    var { data } = await api.get<ExternalResult<DocumentQueryResult<DocumentDetail>>>(`/documents`);
+    setDocumentList(data.payload);
+  }, [api]);
+
+  async function DownloadFile(documentId: number, fileId: number) {
+    var { data } = await api.get<ExternalResult<FileDownload>>(
+      `/documents/${documentId}/files/${fileId}/download`,
+    );
+    const aElement = document.createElement('a');
+    aElement.href = `data:${data.payload.mimetype};base64,` + data.payload.filePayload;
+    aElement.download = data.payload.fileName;
+    aElement.click();
+    //document.body.removeChild(a);
+    window.URL.revokeObjectURL(aElement.href);
+  }
+
+  useEffect(() => {
+    retrieveDocumentList();
+  }, [retrieveDocumentList]);
+
+  const handleFileInput = (e: any) => {
+    console.log(e);
+    // handle validations
+    if (e.target !== null) {
+      var target = e.target as HTMLInputElement;
+      if (target.files !== null) {
+        console.log(target.files[0]);
+        setSelectedFile(target.files[0]);
+      }
+    }
+  };
+
+  const submitForm = () => {
+    if (selectedFile !== null) {
+      const formData = new FormData();
+      formData.append('name', selectedFile.name);
+      formData.append('file', selectedFile);
+
+      api
+        .post(`/documents/`, formData)
+        .then(res => {
+          console.log(res);
```

Comment:
please remove console.logs and alerts

Reply:
Removed logs, but since it is a test page I left the alerts to make sure the result status was successful